.circleci/config.yml

```
@@ -48,7 +50,57 @@ jobs:
       - ms-teams/report:
           only_on_fail: true
           webhook_url: $MS_TEAMS_WEBHOOK_URL
-  build:
+  build-container-python-3-11:
+    docker:
+      - image: cimg/base:stable-20.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build opencti/python-client-container
+          command: |
+            docker run --privileged --rm tonistiigi/binfmt --install all
+            CIRCLE_TAG=${CIRCLE_TAG:-nightly}
+            echo "CIRCLE_TAG=${CIRCLE_TAG}"
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+            docker buildx create --platform linux/amd64,linux/arm64 --use --name mybuilder || true
+            docker buildx inspect mybuilder --bootstrap            
+            docker buildx build . \
+              --platform linux/amd64,linux/arm64 \
+              -t ${{ env.BASE_REPO }}/client-python-3-11:${CIRCLE_TAG} \
+              -t ${{ env.BASE_REPO }}client-python-3-11:latest \
+              --build-arg BASE_IMAGE="python:3.11-alpine3.20" 
+
+      - run:
+          name: Push image to regsitry
+          command: docker push ${{ env.BASE_REPO }}/client-python-3-11:latest --all-tags
+
+  build-container-python-3-12:
+    docker:
+      - image: cimg/base:stable-20.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build opencti/python-client
+          command: |
+            docker run --privileged --rm tonistiigi/binfmt --install all
+            CIRCLE_TAG=${CIRCLE_TAG:-nightly}
+            echo "CIRCLE_TAG=${CIRCLE_TAG}"
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+            docker buildx create --platform linux/amd64,linux/arm64 --use --name mybuilder || true
+            docker buildx inspect mybuilder --bootstrap
+            docker buildx build . \
+              --platform linux/amd64,linux/arm64 \
+              -t ${{ env.BASE_REPO }}/client-python-3-12:${CIRCLE_TAG} \
+              -t ${{ env.BASE_REPO }}/client-python-3-12:latest \
+              --build-arg BASE_IMAGE="python:3.12-alpine3.20" 
+
+      - run:
+          name: Push image to regsitry
```

Could you correct the naming ?

```suggestion
          name: Push image to registry
```